src/main/java/com/purbon/kafka/topology/model/schema/Subject.java

```diff
@@ -14,17 +17,6 @@
   private Optional<String> optionalFormat;
   private SubjectKind kind;
 
-  public enum SubjectKind {
```

Comment:
why the move down of the enum?

Reply:
nothing important really, just ordering, keeping embedded classes at the end.